tests/integration/API/Catalog/Send_Item_Updates/RequestTest.php

```diff
@@ -39,22 +40,54 @@ public function test_constructor() {
 	/** @see Request::set_requests() */
 	public function test_set_requests() {
 
-		$request = new Request( '1234 ');
+		$request  = new Request( '1234 ');
+		$requests = [ [ 'method' => Sync::ACTION_UPDATE ] ];
```

Comment:
Is this not supposed to be indexed by product ID?

```suggestion
		$requests = [ [ '123456' => Sync::ACTION_UPDATE ] ];
```

Reply:
I know it does not affect the test result, but just to avoid any confusion.

Reply:
The `$requests` array here is the one we send to Facebook and will normally include one or more arrays, each with [`retailer_id`, `method`, and `data` entries](https://developers.facebook.com/docs/marketing-api/catalog-batch/guides/send-item-updates#parameters).

Maybe you're referring to the array of requests that we pass to the background job?

I agree that the current approach is confusing though, so I should probably update the arrays in the test to include `retailer_id` and `data` entries too. What do you think?

Reply:
I think that's a good idea, to avoid the same confusion I made, lol!